app/src/main/java/de/westnordost/streetcomplete/data/QuestController.java

```diff
@@ -257,6 +267,53 @@ else if(quest.getStatus() == QuestStatus.CLOSED)
 		});
 	}
 
+	private boolean solveCompleteQuest(long questId, Bundle bundle)
+	{
+		CompleteQuest q = completeQuestDB.get(questId);
+
+		String answer;
+
+		switch (q.getComplete().completeType)
+		{
+			case CompleteTypes.CHART:
```

Comment:
I would really not name this quest type "chart". A chart can be anything, "yesnochart" or just "YesNo" would maybe better represent that.

Reply:
Mmhh... True. I will change this!